logger.gd

```diff
@@ -37,14 +38,14 @@ class Logfile:
 
 	func get_write_mode():
 		if not file.file_exists(path):
-			return File.WRITE # create
+			return File.WRITE  # create
 		else:
-			return File.READ_WRITE # append
+			return File.READ_WRITE  # append
 
 	func validate_path(path):
 		"""Validate the path given as argument, making it possible to write to
 		the designated file or folder. Returns whether the path is valid."""
-		if !(path.is_abs_path() or path.is_rel_path()):
+		if ! (path.is_abs_path() or path.is_rel_path()):
```

Comment:
That's a weird style change. Maybe replace `!` with `not` and then it makes sense.

Reply:
Yes this is still the case and an issue was created: https://github.com/Scony/godot-gdscript-toolkit/issues/106
How do we want to approach this?

Reply:
I'd say work it around with `not` instead of `!`, which is also more pythonic.